packit/utils/changelog_helper.py

```diff
@@ -222,6 +222,14 @@ def prepare_upstream_locally(
         # instead of changing version, we change Release field
         # upstream projects should take care of versions
         if update_release:
+            # Make sure the new release has a dist tag if it is not already included explicitly
+            # (in release) or implicitly (in up.specfile.raw_release)
+            if (
+                release
+                and not release.endswith("%{?dist}")
+                and not self.up.specfile.raw_release.endswith("%{?dist}")
+            ):
+                release = f"{release}%{{?dist}}"
```

Comment:
looks ok to me, but I am not that familiar with this code, maybe @majamassarini @nforro could you take a look please?

Reply:
The code looks good to me but if I get it correctly we are changing the package name format and I think some user may depend on the old format (somehow in their script) and we should be prepared to deal with some issue.
I hope I am wrong.